src/devices/Camera/Camera/Settings/LibcameraAppsSettings.cs

```diff
@@ -0,0 +1,309 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Camera.Settings;
+
+/// <summary>
+/// Settings for all the Libcamera-apps
+/// https://www.raspberrypi.com/documentation/computers/camera_software.html#common-command-line-options
+/// </summary>
+public class LibcameraAppsSettings
+{
+    /// <summary>
+    /// The default options for the Libcamera-apps
```

Comment:
Very minimal: in theory, properties comments should start with "Gets or sets" ("Gets" only in this case")